tests/Architecture/ViewTest.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Illuminate\Contracts\Validation\InvokableRule;
+
+test('view test')
+    ->expect('Cachet\View')
+    ->toBeClasses()
+    ->toExtend(\Illuminate\View\Component::class);
```

Comment:
We can import this class in full.

Reply:
Ooops! I'm not too sure why I didn't import that sorry 🤦‍♂️ That should be fixed now!